src/main/java/moody/Moody.java

```diff
@@ -0,0 +1,71 @@
+package moody;
+
+import moody.commands.Command;
+import moody.exceptions.InvalidCommandException;
+import moody.exceptions.TaskInputException;
+import moody.parser.Parser;
+import moody.storage.Storage;
+import moody.tasks.TaskList;
+import moody.ui.Ui;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Manages the main application for handling tasks, including initializing the task list,
+ * interacting with storage, and processing user commands.
+ */
+public class Moody {
+    private final Storage storage;
+    private final TaskList tasks;
+    private final Ui ui;
```

Comment:
You might want to CAPS LOCK final variables

Reply:
Actually do I have to do that for private final fields as well? My understanding was to do it on private static final fields... I might be wrong. Could I check with you on that?